src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  padding-left: 50px;
+  text-decoration: none;
+}
+
+.nav {
+  display: block;
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  text-transform: uppercase;
+  padding: 0;
+}
+
+.nav__item {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
```

you add margin-left, so remove margin left too
```suggestion
  margin-left: 0;
```